libs/langgraph/tests/test_pregel.py

```diff
@@ -5824,50 +5824,13 @@ def test_entrypoint_from_sync_generator() -> None:
     """@entrypoint does not support sync generators."""
     previous_return_values = []
 
-    @entrypoint(checkpointer=MemorySaver())
-    def foo(inputs, previous=None) -> Any:
-        previous_return_values.append(previous)
-        yield "a"
-        yield "b"
-
-    config = {"configurable": {"thread_id": "1"}}
-
-    assert foo.invoke({"a": "1"}, config) == ["a", "b"]
-    assert previous_return_values == [None]
-    assert foo.invoke({"a": "2"}, config) == ["a", "b"]
-    assert previous_return_values == [None, ["a", "b"]]
-
-
-def test_entrypoint_request_stream_writer() -> None:
-    """Test using a stream writer with an entrypoint."""
-
-    @entrypoint(checkpointer=MemorySaver())
-    def foo(inputs, writer: StreamWriter) -> Any:
-        writer("a")
-        yield "b"
+    with pytest.raises(NotImplementedError):
```

Comment:
nit - maybe mark the test as skipped or just drop it entirely?

Reply:
what behavior do you want if someone attempts to decorator a generator? this test verifies that we raise a not implemented error?

Reply:
ah, I misread - makes sense